### DISCUSSION TITLE
CLI not recognizing absolute path imports?

### DISCUSSION THREAD

**Discussion by jphinning** [tada5hi/typeorm-extension#310] [General]
I'm trying to run a seed with the CLI with the following code:

```typescript
import { Seeder, SeederFactoryManager } from 'typeorm-extension';
import { DataSource } from 'typeorm';
import { Users } from 'src/modules/entities/users.entity';

export default class UserSeeder implements Seeder {
  public async run(
    dataSource: DataSource,
    factoryManager: SeederFactoryManager,
  ): Promise<void> {
    const repository = dataSource.getRepository(Users);
    await repository.insert([
      {
        firstName: 'Caleb',
        lastName: 'Barrows',
        email: 'caleb.barrows@gmail.com',
        password: 'test',
      },
    ]);
}

```

But I'm getting this error:

```
Error: Cannot find module 'src/modules/entities/users.entity'
Require stack:
- ~/src/database/seeds/user.seeder.ts
```
It seems to me that the CLI can't recognize absolute imports. How can I make this work without having to convert all my entities into relative imports?

I'm trying to run the seeds by executing this script in my package.json:
```
"seed": "ts-node ./node_modules/typeorm-extension/dist/cli/index.js seed -- -d src/database/data-source.ts"
```

**Comment by albertcito** [comment]
I have the same problem with nestjs. This is not a solution, but for now I'm running the JS compiled file instead TS files. This is my executing command in my package file:

`ts-node ./node_modules/typeorm-extension/dist/cli/index.js -d ./dist/config/seeder.js seed`

**Reply by tada5hi** [reply]
Does your seeder.js file contain the data-source? 
The `-d <path>` should point to the data-source https://github.com/tada5hi/typeorm-extension#options

**Reply by albertcito** [reply]
It's working properly. The problem is I call from seeder some function from the base code. The base code use absolute path like `@/services/createPost.ts`. That value `@` (defined in `tsconfig.ts`) is unknown for the seeder library.
